cmake/AddResources.cmake

```diff
@@ -45,14 +45,17 @@ function(add_resources target)
 			# of the directory
 			install(DIRECTORY "assets/${dir}"
 				DESTINATION "${INSTALL_APPDATADIR}"
-				COMPONENT "${target}"
 			)
 		endif()
 	endforeach()
 
 	foreach(file IN LISTS ARG_FILES)
 		target_sources(${target} PRIVATE "assets/${file}")
 		set_property(TARGET ${target} APPEND PROPERTY RESOURCE "assets/${file}")
+		get_filename_component(file_dir "${file}" DIRECTORY)
+		set_source_files_properties("assets/${file}"
+			PROPERTIES MACOSX_PACKAGE_LOCATION "Resources${file_dir}"
```

Comment:
This still needs to be using either `set_property(TARGET … APPEND PROPERTY RESOURCE <filename>)` ***or*** `set_source_file_properties(<filename> PROPERTIES MACOSX_PACKAGE_LOCATION)`, not both at the same time.

If `TARGET… RESOURCE` is used instead, that does not support subdirectories, and so there should not be any attempt to do stuff with `${file_dir}` because that won’t work.

Reply:
Another thing I could swear I grepped through and removed. Deleted the TARGET thing.